src/platforms/hosted/README.md

```diff
@@ -1,116 +1,133 @@
-# PC-Hosted BMP
-Compile in src with "make PROBE_HOST=hosted". This needs minimal external
-support.  "make PROBE_HOST=hosted HOSTED_BMP_ONLY=0" will compile support for FTDI,
-STLink, CMSIS-DAP and J-Link probes, but requires external libraries.
+# Black Magic Debug App (BMDA)
+
+Compile with:
+
+```sh
+meson setup build
+meson compile -C build
+```
 
 ## Description
-PC-hosted BMP run on the PC and compiles as "blackmagic". When started,
-it either presents a list of available probes or starts the BMP process
-if either only one probe is attached to the PC or enough information is
-given on the command line to select one of several probes.
-
-When started without any other argument beside the probe selection, a
-GDB server is started on port 2000 and up. Connect to the server as you would
-connect to the BMP with the CDCACM GDB serial server. GDB functionality
-is the same, monitor option may vary.
-
-More arguments allow to
-### Print information on the connected target
+
+BMDA runs on your computer and compiles as `blackmagic`.
+When started, it either presents a list of available probes or starts a GDB server on
+localhost port 2000 if possible (if there's already something bound, it tries the next
+4 port numbers up to 2004), and if there is either only one probe is attached to the
+computer or enough information is given on the command line to select one of several probes.
```

Comment:
```suggestion
When the `blackmagic` executable is launched, it will present a list of available probes.
Alternatively, if there is only one probe attached to the computer, or enough information is given on the
command line to select one of several probes, it will start a GDB server on localhost port 2000
(if the port is already bound, BMDA will try the next 4 port numbers up to 2004).
```